tools/packet_inspector/src/app/hex_viewer.rs

```diff
@@ -47,7 +45,7 @@ impl View for HexView {
 
                     ui.label(format!("{:08X}", count));
                     for b in buf.iter().take(bytes_read) {
-                        ui.colored_label(Color32::from_rgb(255, 255, 255), format!("{:02X}", b));
+                        ui.label(format!("{:02X}", b));
```

Comment:
```suggestion
                        if ui.style_mut().visuals.dark_mode {
                            ui.colored_label(
                                Color32::from_rgb(255, 255, 255),
                                format!("{:02X}", b),
                            );
                        } else {
                            ui.colored_label(Color32::from_rgb(0, 0, 0), format!("{:02X}", b));
                        }
```

the point of the white is so it pops out compared to the ascii representation, something like this will allow it to still do so